setup.py

```diff
@@ -60,8 +60,10 @@
     test_suite='tests',
     zip_safe=zip_safe,
     install_requires=[
+        "email_validator>=1.0.2",
```

Comment:
I believe `email_validator` can also be moved to `test_requires` as well.

Reply:
thanks, fixed in db73eb54